docs/format/source/format_release_notes.rst

```diff
@@ -6,6 +6,11 @@ Release Notes
 2.9.0 (Upcoming)
 ----------------
 
+Major changes
+^^^^^^^^^^^^^
+- Deprecated ``Device.model_number``, ``Device.model_name``, ``Device.serial_number``. Use 
+  ``Device.model`` link and new ``DeviceModel`` neurodata type instead.
```

Comment:
I would add a separate bullet point about _"Added `DevideModel` neurodata type to .... "_ and describe what the purpose is Currently this focuses on what was deprecated but it is not clear what `DeviceModel` is and why it was added. We could also include a link to the issue for further details.

Reply:
done